scripts/mount-cephfs-share.sh

```diff
@@ -34,6 +34,10 @@ function move-rbd-mount {
     if mountpoint ${upgrade_dir} > /dev/null 2>&1; then
       echo "Unmounting ${upgrade_dir}..."
       umount ${upgrade_dir}
+      if [[ "$?" -ne 0 ]]; then
```

Comment:
If the umount fails - what is the remediation?  Should the script to re-run at a later time?

Reply:
the specific case i'm addressing is when the resource is busy (we were in the dir when we umounted) -- and the umount command indicates the failure.  I'll add a note to address the error from umount -- then re-run the script, sound ok?

Reply:
yes